### ISSUE TITLE
Wrong list of dependencies for Fedora 22

### ISSUE THREAD

**Issue by strzibny** [devassistant/dap-ruby#1]
The assistant currently install the following for Rails:

```
- rpm:
  - rubygem(rails)
  - rubygem(sqlite3)
  - rubygem(jquery-rails)
  - rubygem(coffee-rails)
  - rubygem(sass-rails)
  - rubygem(uglifier)
  - rubygem(therubyracer)
```

But for Fedora 22 it should be installing 

```
rubygem-{rails,sqlite3,coffee-rails,sass-rails,uglifier,jquery-rails,turbolinks,jbuilder,therubyracer,sdoc,spring,byebug,web-console}
```

So what's the best way of targeting different Fedora and Rails versions? We have a @rubyonrails group in Fedora, so perhaps it should be installing that. It would be Fedora-version independent.

**Comment by hroncok** [comment]
Thanks for the report. I'm quite confident that with current devassistant, we are not able to install groups. We might try to implement it, but I'm not sure if it will be any time soon.

**Comment by strzibny** [comment]
And btw are there plans on some kind of automatic testing of the assistants? Because we will promote them on the upcoming Fedora developer portal and it's a shame if they don't work as expected...

**Comment by hroncok** [comment]
Currently not. Feel free to propose a way how to do it on the mailing list https://lists.fedoraproject.org/mailman/listinfo/devassistant

We might write tests and run them on Travis, but unfortunately that runs in Ubuntu, so it would not test Fedora packages/dependencies.

**Comment by msrb** [comment]
It looks like DevAssistant recognizes groups:
https://github.com/devassistant/devassistant/blob/master/devassistant/package_managers.py#L279-L280
However, it doesn't seem to work as expected. I will look into it.

@strzibny how can we check that rails app prepared by ruby/rails assistant works correctly? Could something like "bin/rails server" do the trick (for now)?

**Comment by strzibny** [comment]
@msrb yes the first thing to run would be `rails s`, it boots up the whole Rails app that loads all its deps. So if that works, it's most likely okay.
